### ISSUE TITLE
Duplicate error message for instance final field in DeclarationOrder check

### ISSUE THREAD

**Issue by liscju** [checkstyle/checkstyle#2928]
Incorrect position of instance final field is reported once as a instance variable and then as variable in DeclarationOrder. 
For code:

``` java
class Test {
    private int i;
    final int k;
}
```

We have errors: "Instance variable definition in wrong order" and "Variable access definition in wrong order.".
##

**Comment by rnveach** [comment]
I am unable to reproduce with current CS version or older ones.
````
$ cat TestClass.java
class Test {
    private int i;
    final int k;
}

$ cat TestConfig.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
<module name="DeclarationOrder" />
    </module>
</module>

$ java -jar checkstyle-7.6-all.jar -c TestConfig.xml TestClass.java
Starting audit...
[ERROR] TestClass.java:3:5: Variable access definition in wrong order. [DeclarationOrder]
Audit done.
Checkstyle ends with 1 errors.
````

**Comment by liscju** [comment]
I guess this is the result of https://github.com/checkstyle/checkstyle/pull/2920, i think this issue can be closed

**Comment by rnveach** [comment]
@liscju I tried with older versions and I only got at most 1 error message but description says there was 2.
#2920 had one variable assign another, but in this description there are no assignments in example.

Can you recreate the issue? if not than we will close this.

**Comment by rnveach** [comment]
@liscju Have you been able to create the issue on old versions to verify it is truly fixed?

**Comment by rnveach** [comment]
I am closing this as it can't be reproduced. If someone can provide a way to reproduce it, we can re-open the issue.
